src/components/Form.jsx

```diff
@@ -19,7 +19,7 @@ export class Slider extends React.Component {
             step = 1,
             value,
         } = this.props;
-        const barWidth = 120;// css hardcode
+        const barWidth = 238;// css hardcode
```

Comment:
@shayan-binary-pr why is it hardcoded?

Reply:
@morteza-binary actually it has been hardcoded from the beginning, and because of it's calculation. anyway, I can check to use another slider, because as you mentioned, its not good to be hardcode it